src/VolleyManagement.Backend/VolleyManagement.Domain/PlayersAggregate/Player.cs

```diff
@@ -58,7 +57,7 @@ public string LastName
             {
                 if (PlayerValidation.ValidateLastName(value))
                 {
-                    throw new ArgumentException(Resources.ValidationPlayerLastName, "LastName");
+                    throw new ArgumentException(Resources.ValidationPlayerLastName, nameof(Player.LastName));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The parameter name 'LastName' is not declared in the argument list. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3928)